drivers/spi/Kconfig

```diff
@@ -195,6 +211,44 @@ config SPI_2_CS_GPIO_PIN
 	depends on SPI_2 && SPI_CS_GPIO
 	default 0
 
+config SPI_3
+	bool
+	prompt "SPI port 3"
+	default n
+	help
+	  Enable SPI controller port 3.
+
+config SPI_3_NAME
+	string
+	prompt "SPI port 3 device name"
+	depends on SPI_3
+	default "SPI_3"
+
+config SPI_3_IRQ_PRI
+	int
+	prompt "Port 3 interrupt priority"
+	depends on SPI_3
+
+config SPI_3_DEFAULT_CFG
+	hex "Port 3 default configuration"
+	depends on SPI_3
+	default 0x80
+
+config SPI_3_DEFAULT_BAUD_RATE
+	int "Port 3 default baud rate"
+	depends on SPI_3
+	default 500000
+
+config SPI_3_CS_GPIO_PORT
+	string
+	prompt "The GPIO port which is used to control CS"
+	depends on SPI_3 && SPI_CS_GPIO
+	default "GPIO_0"
+
+config SPI_3_CS_GPIO_PIN
+	int "The GPIO PIN which is used to act as a CS pin"
+	depends on SPI_3 && SPI_CS_GPIO
+
```

Comment:
... to this one: remove them.

It's useless. You don't use it and no other code does. And actually, this is legacy stuff that will go away. There is no dflt config (as long as there is no api calls the controller is not running, so there is no need of a configuration) and about the cs/gpio as you can see on the new api it got externalized.